api_tests/lib_sdk/actors/actor.ts

```diff
@@ -360,8 +483,8 @@ function defaultAssetDescriptionForAsset(asset: AssetKind): Asset {
 }
 
 function defaultExpiryTimes() {
-    const alphaExpiry = new Date("2080-06-11T23:00:00Z").getTime() / 1000;
-    const betaExpiry = new Date("2080-06-11T13:00:00Z").getTime() / 1000;
+    const alphaExpiry = Math.round(Date.now() / 1000) + 5;
+    const betaExpiry = Math.round(Date.now() / 1000);
```

Comment:
Beta expires immediately?  Did you mean to add `+ 8`?

Reply:
good point :) 
It's not that bad as we don't refund automatically but looks weird indeed. I'll change it.